assets/home/main.js

```diff
@@ -36,6 +36,10 @@ const Main = () => {
 		fetchAndSetData();
 	}, [] );
 
+	if ( ! data ) {
```

Comment:
It also adds this check to avoid rendering the components without data. It could be a more advanced check and render a spinner, but probably it's not necessary since it's preloaded. :)